bitcoin/src/blockdata/transaction.rs

```diff
@@ -1170,10 +1170,10 @@ impl From<&Transaction> for Wtxid {
 
 /// Computes the value of an output accounting for the cost of spending it.
 ///
-/// The effective value is the value of an output value minus the amount to spend it.  That is, the
-/// effective_value can be calculated as: value - (fee_rate * weight).
+/// The effective value is the value of an output minus the cost to spend it.  That is, the
+/// effective value can be calculated as: value - (fee_rate * weight).
 ///
-/// Note: the effective value of a [`Transaction`] may increase less than the effective value of
+/// Note: the effective value of a [`Transaction`] may increase more than the effective value of
```

Comment:
Nope, this was correct. (It originally wasn't until I pointed it out in my review BTW; it's subtle.)

The logic is inverted - the more bytes transaction has the more you spend on fees thus the value is lower.

Reply:
Oh, that is subtle, I'll improve the docs - thanks.

Reply:
The docs refer to `TxOut` but we are calculating the effective value of in _input_, I didn't notice that before.

Reply:
Well, each input is an output of previous transaction (except for coinbase) so it's often hard to communicate when we kinda mean both.